tests/modules/text_generation/test_peft_tgis_remote.py

```diff
@@ -0,0 +1,101 @@
+"""Tests for prompt tuning based inference via TGIS backend; note that these tests mock the
```

Comment:
Not net-new but renamed from `test_tgis_remote.py` to match source code file (first test is mainly updated to not assume runtime error)